pkg/repository/v1/sqlcv1/olap-overwrite.sql

```diff
@@ -219,3 +219,134 @@ SELECT *
 FROM to_insert
 RETURNING *
 ;
+
+
+-- name: ListEvents :many
+WITH included_events AS (
+    SELECT DISTINCT e.*
```

Comment:
important qq: since runs are now one to many with events, we need to dedupe somehow after joining to the runs. This is the simplest way I could think of, but I'm unsure about performance here - any other ideas? We could e.g. find the first run for each event and then join to that or some such